Automattic/wp-calypso
# Help Center: clean-up article component

Related to #92745

## Proposed Changes

This is a spinoff from the PR mentioned above. This refactors and simplifies the `/post` route that is responsible for displaying support guide articles in the Help Center

## Why are these changes being made?

1. The naming of the components was confusing and misleading
2. There seemed to be unnecessary abstraction
3. Duplicated imports

## Testing Instructions

1. Open Calypso and go to Help Center
2. Test articles and make sure there is no breakage
3. Be sure to try as free user as well to ensure Back button works.
